pkg/base/replicated.go

```diff
@@ -152,7 +157,100 @@ func renderReplicated(u *upstreamtypes.Upstream, renderOptions *RenderOptions) (
 		}
 	}
 
-	return &commonBase, helmBases, nil
+	for i, chart := range kotsHelmCharts {
+		chartBytes, err := yaml.Marshal(chart)
+		if err != nil {
+			return nil, nil, nil, errors.Wrapf(err, "failed to marshal helm chart %s", kotsHelmChartPaths[i])
+		}
+
+		renderedKotsKinds.HelmCharts = append(renderedKotsKinds.HelmCharts, &kotsutil.KotsKindsFile{Path: kotsHelmChartPaths[i], Content: chartBytes})
+	}
+
+	return &commonBase, helmBases, renderedKotsKinds, nil
+}
+
+func renderKotsKinds(upstreamFiles []upstreamtypes.UpstreamFile, renderedConfig *kotsv1beta1.Config, renderOptions *RenderOptions, builder *template.Builder) (*kotsutil.KotsKindsBundle, error) {
+	kotsKinds := kotsutil.KotsKindsBundle{}
```

Comment:
can't this just be a `map[string][]byte`? where the key is the file path and the value is the rendered content?

Reply:
yeah, and it's easier to code that way too, I was just trying to copy the style of the other code that managed file contents 🤦